expect.go

```diff
@@ -159,6 +159,15 @@ func SetEnv(env []string) Option {
 	}
 }
 
+// SetSysProcAttr sets the SysProcAttr syscall values for the spawned process.
```

Comment:
This option will only work in the Spawn* calls. And only really have any effect with the Process related spawners.  Could you add a bit to the documention here explaining it will not work with the GExpect Option method and have little effect for non process spawners.